### DISCUSSION TITLE
How to remove logo from navbar.

### DISCUSSION THREAD

**Discussion by alexmpaschal** [timlrx/tailwind-nextjs-starter-blog#414] [Q&A] [answered]
I don't have a logo that I want to use for the navbar and was wondering how I can remove that component and just have the title there. Thanks in advance.

**Comment by timlrx** [comment, marked as answer]
Just remove the logo directly from the [LayoutWrapper component](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/components/LayoutWrapper.js#L19).
